web/styles/zulip.css

```diff
@@ -1459,6 +1456,24 @@ a.dark_background:hover,
     color: hsl(200deg 99% 60%);
 }
 
+/* UI redesign: blue links style: https://github.com/zulip/zulip/issues/24877 */
```

Comment:
You can remove the comment -- https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#commit-description touches on this briefly, but comments in the code should explain the code to someone reading the post-commit code, and the issue link is really only relevant to understanding the change.

Reply:
okay, I'll get to it right away